x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/components/getting_started/getting_started.tsx

```diff
@@ -98,7 +100,7 @@ export const APIGettingStarted = () => {
               language={language}
               setSelectedLanguage={setSelectedLanguage}
               isSelectedLanguage={selectedLanguage === language}
-              http={http}
+              assetBasePath={assetBasePath}
```

Comment:
Is it necessary to pass for dedicated as we pass the `src` prop. ( which I have named really poorly)

Reply:
correct I'll remove this. I think I was replacing usages with find.